.circleci/config.yml

```diff
@@ -189,6 +189,7 @@ jobs:
       USE_GANACHE_DEPLOYMENT_CACHE: true
       GANACHE_CACHE_FOLDER: ../../.ganache-deployments
       GANACHE_PORT: 8547
+      NODE_ENV: test
```

Comment:
I don't think this is needed since `NODE_ENV` is set to `test` in CircleCI project environment variables: https://circleci.com/gh/statechannels/monorepo/edit#env-vars

Reply:
I added this because for anyone who in the future forks this repo will be tripped by this CircleCI settings like I did. 

Ideally,. all vars should be in this file (except maybe secrets?) so nobody needs to tinker with CircleCI. We have lots of env vars in `config.yml` anyway, why not add one extra.

Reply:
@junjizhi you bring up a good point. I am ok defining env vars in `config.yml` with a couple caveats:
- An env variable should be defined either in `config.yml` or in the CircleCI admin console. But not in both.
- If we are moving away from defining env variables in CircleCI admin console (except for secrets), we should create reasonable defaults that most jobs inherit. A quick scan of `config.yml` shows that we might need to do a bit of clean up with env vars and override defaults only when needed.

Reply:
@kerzhner I totally agree with the above two points. Having the setting in both places is bad for maintenance and newcomers. 

I don't have access to CircleCI admin, so I couldn't see env var settings there, so no way for me to compare with `config.yml` and tell 1) what are the defaults, 2) what gets overridden there. The cleanup may have to be done in a separate ticket.

Reply:
Sounds excellent, filed https://github.com/statechannels/monorepo/issues/1054